hbase-client/src/test/java/org/apache/hadoop/hbase/TestCatalogFamilyFormat.java

```diff
@@ -70,4 +79,32 @@ public void testMetaReaderGetColumnMethods() {
         HConstants.SEQNUM_QUALIFIER_STR + CatalogFamilyFormat.META_REPLICA_ID_DELIMITER + "002A"),
       CatalogFamilyFormat.getSeqNumColumn(42));
   }
+
+  /**
+   * The info we can get from the regionName is: table name, start key, regionId, replicaId.
+   */
+  @Test
+  public void testparseRegionInfoFromRegionName() throws IOException  {
+    TableName tableName = TableName.valueOf(name.getMethodName());
+    long regionId = 999999L;
```

Comment:
NIT: As these variables are only used in the builder (and with that their purpose is also visible) below they can be directly used in the builder.